docs/README.md

```diff
@@ -1,11 +1,23 @@
 **Models of some of the HW present in a NRF52xxx.**<br>
 Where relevant differences exist, these models try to align with a NRF52382.
 
-This repo contains both models of the NRF52 HW as well as a replacement nRFx
-HAL.<br>
-This replacement HAL is a modified version of the real nRFx to be used with
-these models.
-With it, the Zephyr SW should work without needing further changes.
+This repo models the nRF peripheral by implementing of the nrfx HAL functions.
```

Comment:
I missed this sentence when reviewing before (there is a "grammar typo" :) ) 
Anyhow, would it make sense to replace with
```
This repo contains both models of the NRF52 HW as well as some replacement nrfx
HAL functions. When used in combination with the real nrfx, these should enable code
meant for the nrfx to run without needing further changes.
This includes Zephyr SW.
```
?